qutip/floquet.py

```diff
@@ -100,15 +99,11 @@ def floquet_modes(H, T, args=None, sort=False, U=None):
         U = propagator(H, T, [], args)
 
     # find the eigenstates for the propagator
-    evals, evecs = la.eig(U.full())
+    evals, evecs = U.eigenstates()
 
     eargs = angle(evals)
-
-    # make sure that the phase is in the interval [-pi, pi], so that
-    # the quasi energy is in the interval [-pi/T, pi/T] where T is the
-    # period of the driving.  eargs += (eargs <= -2*pi) * (2*pi) +
-    # (eargs > 0) * (-2*pi)
-    eargs += (eargs <= -pi) * (2 * pi) + (eargs > pi) * (-2 * pi)
+    # note: angle is in the interval [-pi, pi], so that the quasi energy is in
+    # the interval [-pi/T, pi/T] where T is the period of the driving.
```

Comment:
The comment was misleading, using `2 * pi` whereas the actual line below was using a mix of `pi` and `2*pi`.

From https://docs.scipy.org/doc/numpy-1.13.0/reference/generated/numpy.angle.html and https://docs.scipy.org/doc/numpy-1.13.0/reference/generated/numpy.arctan2.html#numpy.arctan2, it seems that the value returned by `angle` is always in the range `[-pi, pi]` anyways, so I guess there should not be any need for this correction.